packages/hoprd/crates/hopr-lib/src/processes/mod.rs

```diff
@@ -1,77 +1,134 @@
+                                let _ = core_strategy::strategy::SingularStrategy::on_own_channel_changed(
+                                    &*multi_strategy,
+                                    &channel,
+                                    own_channel_direction,
+                                    channel_change,
+                                )
+                                .await;
 
-                        // Cleanup invalid tickets from the DB if epoch has changed
-                        // TODO: this should be moved somewhere else once event broadcasts are implemented
-                        if let core_types::channels::ChannelChange::Epoch { .. } = channel_change {
-                            let _ = db.write().await.cleanup_invalid_channel_tickets(&channel).await;
+                                // Cleanup invalid tickets from the DB if epoch has changed
+                                // TODO: this should be moved somewhere else once event broadcasts are implemented
+                                if let core_types::channels::ChannelChange::Epoch { .. } = channel_change {
+                                    let _ = db.write().await.cleanup_invalid_channel_tickets(&channel).await;
+                                }
+                            }
+                        } else if channel.status == core_types::channels::ChannelStatus::Open {
+                            // Emit Opening event if the channel did not exist before in the graph
+                            let _ = core_strategy::strategy::SingularStrategy::on_own_channel_changed(
+                                &*multi_strategy,
+                                &channel,
+                                own_channel_direction,
+                                core_types::channels::ChannelChange::Status {
+                                    left: core_types::channels::ChannelStatus::Closed,
+                                    right: core_types::channels::ChannelStatus::Open,
+                                },
+                            )
+                            .await;
                         }
                     }
-                } else if channel.status == core_types::channels::ChannelStatus::Open {
-                    // Emit Opening event if the channel did not exist before in the graph
-                    let _ = core_strategy::strategy::SingularStrategy::on_own_channel_changed(
-                        &*multi_strategy,
-                        &channel,
-                        own_channel_direction,
-                        core_types::channels::ChannelChange::Status {
-                            left: core_types::channels::ChannelStatus::Closed,
-                            right: core_types::channels::ChannelStatus::Open,
-                        },
-                    )
-                    .await;
-                }
+                },
+                SignificantChainEvent::NetworkRegistryUpdate(address, allowed) => {
```

Comment:
When is this called? Can it be removed?

Reply:
This is an event that gets raised once an address is added/removed from the Network Registry, right ?